packages/playground/blueprints/src/lib/steps/run-sql-file.ts

```diff
@@ -0,0 +1,69 @@
+import { StepHandler } from '.';
+
+/**
+ * @inheritDoc runSqlFile
+ * @hasRunnableExample
+ * @example
+ *
+ * <code>
+ * {
+ * 	    "step": "runSqlFile",
+ * 		"path": "/tmp/my-backup.sql"
+ * }
+ * </code>
+ */
+export interface RunSqlFileStep {
+	/**
+	 * The step identifier.
+	 */
+	step: 'runSqlFile';
+	/**
+	 * The SQL file to run.
+	 */
+	path: string;
```

Comment:
this is still out of date with the `runSqlFile` function below, which means it's not explaining in the schema what kind of `path` it is.

would it be right to assume that this is a path into the Playground for an asset that should have already been uploaded with another step by this point?

```ts
/**
 * Path into Playground's virtual filesystem where an SQL
 * file should be present by the time this step runs.
 *
 * The SQL file should be a plaintext file with raw SQL
 * commands separated by newline characters. The contents
 * are fed directly into `$wpdb->query()`.
 */
```

Reply:
@dmsnell Sorry about that, we're planning to roll this & RunSqlQueryStep into one thing that will accept a `Resource`, so as not to have duplicated code. Updates soon!

Reply:
@dmsnell  This file has been removed.

https://github.com/WordPress/wordpress-playground/pull/745/files